.idea/codeStyles/Project.xml

```diff
@@ -5,6 +5,22 @@
       <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999" />
     </JavaCodeStyleSettings>
     <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
```

Comment:
These files are auto-generated by IntelliJ. They don't have to be included in your PR.
Check https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR for more info on the guidelines. Also, please self-review once, before checking the checklist while creating a PR.